File: app/Http/Controllers/Lookup/SpeciesController.php
<?php

namespace App\Http\Controllers\Lookup;

use App\Http\Controllers\Controller;
use Illuminate\Http\Request;
use Illuminate\View\View;

class SpeciesController extends Controller
{
    public function index(): View
    {
        return view('specimens.look_up_tables.species');
    }

    public function create() {}

    public function store(Request $request) {}

    public function show($id) {}

    public function edit($id) {}

    public function update(Request $request, $id) {}

    public function destroy($id) {}
}

File: app/controllers/admin/species_controller.rb
class Admin::SpeciesController < Admin::ApplicationController
  before_action :set_species, only: %i[show edit update destroy]

  # GET /species
  def index
    authorize Species
    @species = policy_scope(Species.order(:taxon_name))
  end

  # GET /speciess/1
  def show
    authorize @species
  end

  # GET /speciess/new
  def new
    @species = Species.new
    authorize @species
  end

  # POST /speciess
  def create
    @species = Species.new(species_params)
    authorize @species
    if @species.save
      redirect_to admin_species_path(@species), notice: "Species was successfully created."
    else
      render :new, status: :unprocessable_entity
    end
  end

  # GET /speciess/1/edit
  def edit
    authorize @species
  end

  # PATCH/PUT /speciess/1
  def update
    authorize @species
    if @species.update(species_params)
      redirect_to admin_species_path(@species), notice: "Species was successfully updated."
    else
      render :edit, status: :unprocessable_entity
    end
  end

  # DELETE /speciess/1
  def destroy
    authorize @species
    @species.destroy!
    redirect_to admin_species_path, notice: "Species was successfully destroyed."
  end

  private

  def set_species
    @species = Species.find(params.expect(:id))
  end

  def species_params
    params.expect(species: [:mb_list_id, :taxon_name, :genera_id])
  end
end
